src/chrome/content/rules/LLVM.org.xml

```diff
@@ -30,24 +27,12 @@ Fetch error: http://www.reviews.llvm.org/ => https://www.reviews.llvm.org/: (6,
 	for reviews
 
 -->
-<ruleset name="LLVM.org (partial)" default_off='failed ruleset test'>
+<ruleset name="LLVM.org (partial)">
 
 	<!--	Direct rewrites:
 				-->
 	<target host="llvm.org" />
 	<target host="reviews.llvm.org" />
-	<target host="*.reviews.llvm.org" />
-
-		<!--	includeSubdomains applies to one level only, so:
-									-->
-		<exclusion pattern="^http://(?:[^./]+\.){2,}reviews\.llvm\.org/" />
-
-			<!--	+ve:
-					-->
-			<test url="http://this.host.reviews.llvm.org/" />
-			<test url="http://exists.not.reviews.llvm.org/" />
-
-		<test url="http://www.reviews.llvm.org/" />
 
 	<!--	Complications:
 				-->
```

Comment:
It seems like the complication rules can be replaced with direct rewriting rules now